packages/vue/src/examples/pages/home/Home.vue

```diff
@@ -32,7 +32,10 @@
       image="/assets/storybook/Home/newsletter.jpg"
     />
     <SfSection title-heading="Bestsellers">
-      <SfCarousel class="carousel">
+      <SfCarousel 
+        :settings="isMobile ? { peek: 0 } : { peek: 16 }" 
```

Comment:
@justyna-13 maybe https://glidejs.com/docs/options/#breakpoints be better than isMobile

Reply:
breakpoints added and also perView option. Now, on mobile it looks similarly to the design on figma. Wdyt, @pspaczek ? :)